notebooks/feature_engineering/solutions/4_keras_adv_feat_eng.ipynb

```diff
@@ -31,15 +31,23 @@
    "metadata": {},
```

Comment:
<p>Let's remove the line here which says, "Note that the dataset is the same as used in the Big Query feature engineering labs.&nbsp;" We rarely do the BQML lab before this one. This way the lab can be independent.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/197/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>